packages/mediacenter/LibreELEC-settings/scripts/factory-reset

```diff
@@ -71,19 +75,15 @@ if [ -f /storage/.cache/reset_oe ] ; then
         tune2fs -U $uuid $part 2>&1 >/dev/null
       fi
     fi
```

Comment:
You've removed the `StopProgress` call, which you need here as you're not passing the command to be run as the third argument, so the spinner never knows when to finish spinning.

I'd actually move the `StartProgress`/`StopProgress` calls to be within the `if [ -n "$part_start" ] ; then` condition - if you've not got a valid `$part_start` variable then there's no point even starting the spinner (you might instead want to show the user a useful/informative error message should this situation ever arise, rather than fail silently).